news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,211 @@
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Tenshichan.jpg)](https://osu.ppy.sh/users/1101600)
+
+#### Kamikaze
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Kamikaze.jpg)](https://osu.ppy.sh/users/2124783)
+
+#### Pope Gadget
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Pope-Gadget.jpg)](https://osu.ppy.sh/users/2288341)
+
+## The Community Mentorship Program
+
+Founded in July 2016, the Community Mentorship Program has run for over two years at this point, helping hundreds of novice mappers find experienced creators to help guide their learning of an increasingly difficult and complicated craft.
+
+We thus, commend the immense contribution made by those who have helped and continue to help run this project, for managing such a large group of people so consistently for such a long period of time is absolutely no small feat.
+
+### Yuii-
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Yuii.jpg)](https://osu.ppy.sh/users/2935923)
+
+Responsible for building the foundations of the project and an acclaimed 2-year+ Beatmap Nominator to boot, Yuii-'s presence in the Mentorship Program has been instrumental in its creation and continued existence.
+
+### Halfslashed
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Halfslashed.jpg)](https://osu.ppy.sh/users/4598899)
+
+Very much the forward 'face' of the project, Halfslashed has been deep in the thick of the social aspects of leadership, being largely responsible for the deluge of speedmapping and other mapping contests that form the bulk of the competitive scene within the program.
+
+### Mir
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Mir.jpg)](https://osu.ppy.sh/users/8688812)
+
+An effortless generalist and a former mentee of the program himself under Shiirn, Mir's presence in the project was confined to no one particular domain, but showcased his ability to coordinate and implement at a level that would later speak well in his favor for his entry into the Quality Assurance Team.
+
+### Mun
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Mun.jpg)](https://osu.ppy.sh/users/6699165)
+
+Once a former mentee himself under Shiirn, Mun's efforts have spanned across dozens of domains within the project, primarily handling moderation and evaluation related tasks pertaining to both mentors and mentees.
+
+### J1NX1337
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/J1nx1337.jpg)](https://osu.ppy.sh/users/3971179)
+
+Responsible for handling much of the busywork deluge that often accompanies managing a project of such size and consistency, J1NX1337's efforts were instrumental to the continuation of the project, and while not all work is exactly glamorous, such things are certainly no less important towards making things happen.
```

Comment:
Fairly minor but `deluge` is used for the second time (in the Mentorship section alone). Just removing it here should be fine

Reply:
also fixed